<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="DESKTOP-CE8NA49" failures="1" tests="2" name="edge" time="133.769" errors="0" timestamp="2023-03-16T22:47:50 IST">
  <testcase classname="testClassPackage.TestClassTNG2" name="VerifyLoginFunctionalityWithCorrectData" time="9.387"/>
  <testcase classname="testClassPackage.TestClassTNG2" name="wrongUserNamePassword" time="0.013">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testClassPackage.TestClassTNG2.wrongUserNamePassword(TestClassTNG2.java:51)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- wrongUserNamePassword -->
  <testcase classname="testClassPackage.BaseClass" name="@AfterTest afterTest" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterTest afterTest -->
  <testcase classname="testClassPackage.BaseClass" name="@BeforeTest beforeTest" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeTest beforeTest -->
</testsuite> <!-- edge -->
